app/src/utils/fetch-user-perms.ts

```diff
@@ -1,5 +1,6 @@
 import api, { route } from '@forge/api';
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Are we going to be using this in future?

Reply:
Good catch, this disable doesnt need to exist